.github/workflows/scheduled-arm64.yml

```diff
@@ -0,0 +1,11 @@
+name: Post-Validation on Linux ARM64
+
+on:
+  schedule:
+    - cron: "*/2 * * * *"
```

Comment:
Every 2 minutes to be able to see the result quickly. Once everything works the schedule will be once per day or something similar.